.github/workflows/package-helm.yaml

```
@@ -30,10 +35,79 @@ jobs:
             mv $SRC_FILE $DEST_FILE
           done
 
-      - name: Upload Helm packages to GitHub releases
+      - name: Upload Release Helm Chart Packages to GitHub Release
         uses: softprops/action-gh-release@v1
         with:
           name: ${{ github.ref_name }}
           draft: true
           append_body: true
           files: nri-*helm-chart*.tgz
+
+  unstable:
+    concurrency:
+      group: gh-pages
+      cancel-in-progress: false
+    if: ${{ github.env.IS_RELEASE != 'true' }}
```

I think you can simplify it a little 
```suggestion
    if: ${{ !github.env.IS_RELEASE }}
```

---

I don't think that would work. I think everything in `github.env`  is unconditionally a `string`, even `'true'` and `'false'`... At least I could not find by a quick search any conditionals that would evaluate `github.env.*` directly as a boolean.